examples/deck.gl/pointcloud/file-index.js

```diff
@@ -14,6 +14,9 @@ export default {
     },
     'Bun Zipper (Text)': {
       uri: `${LOADERS_URI}/modules/ply/test/data/bun_zipper.ply`
+    },
+    'Richmond Azaelias': {
```

Comment:
That is a beautiful cloud!

Note that each module's `test/data` folder has a README that lists the licenses of any data files we copy into this repository. Just make sure this file is under a permissive license and that this is stated in that README, to make sure there are no licensing concerns with merging this PR.

Reply:
very good point! I've added it now